src/cute_shader/cute_shader.h

```diff
@@ -0,0 +1,78 @@
+#ifndef CUTE_SHADER_H
+#define CUTE_SHADER_H
+
+#include <stddef.h>
+#include <cute_shader_bytecode.h>
+
+typedef struct cute_shader_define_t {
```

Comment:
Could we use CF_ style for structs, and brace on separate line

Reply:
Sure, for cute shader, it's not really part of CF though. How should I name it:

* `CF_ShaderDefine`
* `CSH_Define`: CuteShader define.
* `CS_Define`: But `CS` is already `cute_sound`

Reply:
I'd say just go with CF_ prefix